packages/client/hmi-client/src/workflow/ops/calibrate-julia/tera-calibrate-node-julia.vue

```diff
@@ -1,398 +1,60 @@
-			</AccordionTab>
-			<AccordionTab header="Calibrated parameter values">
-				<table class="p-datatable-table">
-					<thead class="p-datatable-thead">
-						<th>Parameter</th>
-						<th>Value</th>
-					</thead>
-					<tr v-for="(content, key) in parameterResult" :key="key">
-						<td>
-							<p>{{ key }}</p>
-						</td>
-						<td>
-							<p>{{ content }}</p>
-						</td>
-					</tr>
-				</table>
-			</AccordionTab>
-			<AccordionTab header="Variables">
-				<tera-calibrate-chart
-					v-for="(cfg, index) of node.state.calibrateConfigs.chartConfigs"
-					:key="index"
-					:initial-data="csvAsset"
-					:intermediate-data="currentIntermediateVals"
-					:mapping="mapping"
-					:chartConfig="{ selectedRun: runInProgress!, selectedVariable: cfg }"
-					@configuration-change="chartConfigurationChange(index, $event)"
-				/>
-				<Button
-					class="add-chart"
-					text
-					:outlined="true"
-					@click="addChart"
-					label="Add chart"
-					icon="pi pi-plus"
-				></Button>
-			</AccordionTab>
-		</Accordion>
 	</section>
 </template>
 
 <script setup lang="ts">
-import { computed, shallowRef, watch, ref, onMounted, onUnmounted } from 'vue';
+import { ref, onMounted, onUnmounted } from 'vue';
 import { ProgressState, WorkflowNode } from '@/types/workflow';
-import DataTable from 'primevue/datatable';
-import Button from 'primevue/button';
-import Column from 'primevue/column';
-import Accordion from 'primevue/accordion';
-import AccordionTab from 'primevue/accordiontab';
 import {
-	CalibrationRequestJulia,
-	ClientEvent,
-	ClientEventType,
-	CsvAsset,
-	ModelConfiguration,
-	ScimlStatusUpdate
-} from '@/types/Types';
-import {
-	makeCalibrateJobJulia,
-	getRunResultJulia,
 	simulationPollAction,
-	querySimulationInProgress,
-	subscribeToUpdateMessages,
-	unsubscribeToUpdateMessages
+	querySimulationInProgress
 } from '@/services/models/simulation-service';
-import { setupModelInput, setupDatasetInput, renderLossGraph } from '@/services/calibrate-workflow';
-import { ChartConfig, RunResults, RunType } from '@/types/SimulateConfig';
-import { csvParse } from 'd3';
-import _ from 'lodash';
-import InputNumber from 'primevue/inputnumber';
-import InputText from 'primevue/inputtext';
-import Dropdown from 'primevue/dropdown';
 import { Poller, PollerState } from '@/api/api';
-import TeraSimulateChart from '@/workflow/tera-simulate-chart.vue';
-import TeraCalibrateChart from '@/workflow/tera-calibrate-chart.vue';
 import TeraProgressBar from '@/workflow/tera-progress-bar.vue';
-import { getTimespan } from '@/workflow/util';
 import { logger } from '@/utils/logger';
-import {
-	CalibrationOperationJulia,
-	CalibrationOperationStateJulia,
-	CalibrateMap,
-	CalibrateMethodOptions,
-	CalibrateExtraJulia
-} from './calibrate-operation';
+import { CalibrationOperationStateJulia } from './calibrate-operation';
+
+// TODO: FIGURE OUT WHAT TO SHOW IN THIS WORKFLOW NODE WHILE CALIBRATION IS RUNNING FROM THE DRILLDOWN
```

Comment:
Should be safe to remove this. Every single node that can be run will have this same question answered this will not be a unique issue
If it is unique enough to track we should create a github issue rather than leaving this TODO as this todo will be lost to time

Reply:
alrighty removed